### ISSUE TITLE
Pin tool versions

### ISSUE THREAD

**Issue by bitcoinhodler** [jazarija/icecube#5]
To reduce risk of third-party package malware (see [GlacierProtocol#42](https://github.com/GlacierProtocol/GlacierProtocol/issues/42)) the `apt-get install` on line 34 should request specific versions of each package.

This will also likely reduce maintenance work required on Glacier if these packages should make incompatible changes in the future.

**Comment by jazarija** [comment]
Agreed. I'll make a global variable defining which versions of zbarcam and qrencode should be downloaded by apt.

**Comment by bitcoinhodler** [comment]
Why only those two? What about all the Ubuntu infrastructure?

Today's Glacier achieves this pinning by using 16.04.1 specifically.

**Comment by jazarija** [comment]
You're right. I'll pin to specific versions as soon as we get the base system in a workable way as needed by the protocol.
